### ISSUE TITLE
[Accessibility Bug] Privacy Agreement modal tabs to "blank" element

### ISSUE THREAD

**Issue by AminaSalah** [CDH-Studio/UpSkill#1544]
**Describe the bug**
Privacy Agreement Modal:
When tabbing through the privacy agreement modal with NVDA, I can tab to the language switcher -> decline btn -> accept btn ->“blank”. I’m not sure what the “Blank” is, but it should either be more descriptive of what it is or removed if unnecessary.

**Comment by MenaMachado** [comment]
Unable to fix -remove - it's an ant design issue.
